### DISCUSSION TITLE
How to resize the pictures in curator picker ?

### DISCUSSION THREAD

**Discussion by nakir00** [awcodes/filament-curator#331] [Q&A]
I've just installed curator and I've been able to add images, but the problem arises when I use the curator picker: the images are too big,
see next image :

![image](https://github.com/awcodes/filament-curator/assets/105824977/98a17a52-faf3-4524-86e5-aec903b7f8a1)

i'd like to be able to change the image display size.

**Comment by awcodes** [comment]
Looks like you're not getting the styles. Did you follow the steps for creating a custom theme adding Curator to that theme?

**Reply by nakir00** [reply]
I imported it into app/ressources/css/app.css as shown in the image:

![image](https://github.com/awcodes/filament-curator/assets/105824977/54dc4fba-7ce1-4a45-96d7-8c3dbab6c68e)

I think this is the right file, isn't it ?🤔

**Reply by ArtMin96** [reply]
@nakir00 [Creating a custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme) Do these steps
